integration-cli/docker_cli_run_test.go

```diff
@@ -2538,3 +2538,10 @@ func (s *DockerSuite) TestRunNetworkFilesBindMount(c *check.C) {
 		c.Fatalf("expected resolv.conf be: %q, but was: %q", expected, actual)
 	}
 }
+
+func (s *DockerSuite) TestRunCapAddSYSTIME(c *check.C) {
+	out, _ := dockerCmd(c, "run", "--cap-drop=ALL", "--cap-add=SYS_TIME", "busybox", "sh", "-c", "date --set `date '+%H:%M:%S'` > /dev/null && sleep 1 && echo ok")
```

Comment:
@sunyuan3 sorry I just noticed: would you mind removing `&& sleep 1 && echo ok` ? You only need to really check the err returned by dockerCmd. If there was an error make sure you provide the output `out` for easier debugging. Thanks!